types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
This is especially convincing to me because I am not making any claims that _all_ the types for 0.17.2 are there - I am just adding the one I ran across that wasn't. But this is my first pr into dt so I'm hoping someone knows better and can tell me for sure :)